Cisco/cisco-meraki/ingest/parser.yml

```
@@ -0,0 +1,306 @@
+  set_logsubtype:
+    actions:
+      - set:
+          cisco_meraki.event_subtype: '{{grok_events.message.log_subtype}}'
+        filter: ''
+        name: set
+  set_dhcp:
+    actions:
+      - set:
+          client.ip: '{{grok_dhcp.message.client_ip}}'
+          client.mac: '{{grok_dhcp.message.client_mac}}'
+          server.mac: '{{grok_dhcp.message.server_mac}}'
+          cisco_meraki.event_subtype: dhcp_offer
+        filter: '{{grok_dhcp.message.dhcp_op == "lease"}}'
+        name: set
+      - set:
+          client.mac: '{{grok_dhcp.message.client_mac}}'
+          cisco_meraki.event_subtype: dhcp_no_offer
+        filter: >-
+          {{ grok_dhcp.message.dhcp_op == "no" and grok_dhcp.message.dhcp_op2 ==
+          "offers"}}
+        name: set
+  set_anyconnect_vpn:
+    actions:
+      - set:
+          client.ip: '{{grok_anyconnect_vpn.message.client_ip}}'
+          server.ip: '{{grok_anyconnect_vpn.message.server_ip}}'
+          client.port: '{{grok_anyconnect_vpn.message.client_port}}'
+          server.port: '{{grok_anyconnect_vpn.message.server_port}}'
+          network.protocol: '{{grok_anyconnect_vpn.message.protocol}}'
+        filter: ''
+        name: set
+  set_content_filtering_block:
+    actions:
+      - set:
+          server.ip: '{{grok_content_filtering_block.message.server_ip}}'
+          client.mac: '{{grok_content_filtering_block.message.client_mac}}'
+          server.port: '{{grok_content_filtering_block.message.server_port}}'
+          url.original: '{{grok_content_filtering_block.message.url}}'
+          action.outcome: block
+        filter: ''
+        name: set
+  set_anyconnect_vpn_connect:
+    actions:
+      - set:
+          network.forwarded_ip: '{{grok_anyconnect_vpn_connect.message.network_forwarded_ip}}'
+          client.ip: '{{grok_anyconnect_vpn_connect.message.client_ip}}'
+          user.name: '{{grok_anyconnect_vpn_connect.message.user_name.split(''@'')[0]}}'
+        filter: ''
+        name: set
+  set_anyconnect_vpn_session_manager:
+    actions:
+      - set:
+          client.ip: '{{grok_anyconnect_vpn_session_manager.message.client_ip}}'
+          user.name: >-
+            {{grok_anyconnect_vpn_session_manager.message.user_name.split('@')[0]}}
+          cisco_meraki.end_of_message: '{{grok_anyconnect_vpn_session_manager.message.end_of_message}}'
+        filter: ''
+        name: set
+  set_anyconnect_vpn_auth:
+    actions:
+      - set:
+          action.outcome: '{{grok_anyconnect_vpn_auth.message.outcome}}'
+          action.outcome_reason: '{{grok_anyconnect_vpn_auth.message.end_of_message}}'
+          client.ip: '{{grok_anyconnect_vpn_auth.message.client_ip}}'
+          client.port: '{{grok_anyconnect_vpn_auth.message.client_port}}'
+          server.ip: '{{grok_anyconnect_vpn_auth.message.server_ip}}'
+          server.port: '{{grok_anyconnect_vpn_auth.message.server_port}}'
+          user.name: '{{grok_anyconnect_vpn_auth.message.user_name.split(''@'')[0]}}'
```

You can extract the domain of the user too.
```suggestion
          user.name: '{{grok_anyconnect_vpn_auth.message.user_name.split(''@'')[0]}}'
          user.domain: '{{grok_anyconnect_vpn_auth.message.user_name.split(''@'')[1]}}'
```